### DISCUSSION TITLE
Display Resolution setup in dotnet/framework/sdk:4.8-windowsservercore-ltsc2019 windows server core docker image to 1920X1080

### DISCUSSION THREAD

**Discussion by RavitejaAmruthapu** [microsoft/dotnet-framework-docker#1165] [Q&A]
Hi, 

How to setup the dotnet/framework/sdk:4.8-windowsservercore-ltsc2019 windows server core docker container display resolution to 1920X1080?

please help me to resolve this issue

Thanks & Regards,
Raviteja

**Comment by lbussell** [comment]
Please open this discussion on the [microsoft/Windows-Containers](https://github.com/microsoft/Windows-Containers/discussions) repo, since your question applies to the Windows base image rather than any .NET-specific features.

**Reply by RavitejaAmruthapu** [reply]
thanks for replay. sure will do
